src/app/portfolio/portfolio.component.ts

```diff
@@ -1,92 +1,96 @@
 import {ChangeDetectionStrategy, Component} from '@angular/core'
-import { BehaviorSubject } from 'rxjs'
-import { map } from 'rxjs/operators'
+import {BehaviorSubject} from 'rxjs'
+import {map} from 'rxjs/operators'
 
 @Component({
-  selector: 'app-portfolio',
-  templateUrl: './portfolio.component.html',
-  styleUrls: ['./portfolio.component.css'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
+    selector: 'app-portfolio',
+    templateUrl: './portfolio.component.html',
+    styleUrls: ['./portfolio.component.css'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class PortfolioComponent {
+    projectStateType = ProjectState
```

Comment:
You can call it just `projectState` as there is no other variable that could share the same name. The same thing for `portfolioModel$`; you can just call it `portfolio` as it's a base source for portfolio, any other portfolio-related variable should have a suffix, like `portfolioSub` that indicates it's a portfolio subscription. 
I've adopted this principle from somewhere and I like it because it's clear.